include/nbl/video/surface/ISurfaceVK.h

```diff
@@ -17,13 +17,240 @@ class ISurfaceVK : public ISurface
 
     bool isSupported(const IPhysicalDevice* dev, uint32_t _queueIx) const override;
 
-protected:
-    ISurfaceVK() = default;
+    static inline asset::E_FORMAT getFormat(VkFormat in)
```

Comment:
this needs to move to some general Vulkan utility header (whatever the top level thing providing `volk.h` is)


You'll need it all over the place